specification/ai/OpenAI.Assistants/files/routes.tsp

```diff
@@ -81,3 +81,13 @@ op getFile(
   @encodedName("application/json", "file_id")
   fileId: string,
 ): OpenAIFile;
+
+@get
+@route("/files/{fileId}/content")
```

Comment:
Yeah. I also checked the openai yaml file, seems we are missing this 

https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L1549

Reply:
From what I can tell this is the right TypeSpec - my hope is that this generates a method for each client that returns a stream, or similar, for the body contents.